pyqtgraph/parametertree/parameterTypes/color.py

```diff
@@ -1,6 +1,7 @@
 from ... import functions as fn
 from ...widgets.ColorButton import ColorButton
 from .basetypes import SimpleParameter, WidgetParameterItem
+from qtpy import QtGui
```

Comment:
please don't use qtpy, but in any case, QtGui doesn't seem to be needed?

Reply:
Ok I change it